crates/types/src/config/networking.rs

```diff
@@ -27,6 +27,12 @@ pub struct NetworkingOptions {
     ///
     /// Retry policy to use for internal node-to-node networking.
     pub connect_retry_policy: RetryPolicy,
+
+    /// # Handshake timeout
+    ///
+    /// Time a node waits for handshake message to be received before
+    /// giving up
+    pub handshake_timeout: Duration,
```

Comment:
Can we use `humantime::Duration` instead and setup schemar's correctly for it? You can find an example in `config/http.rs`

Reply:
definitely!